src/models/tools/tool-types.ts

```diff
@@ -28,12 +29,16 @@ export type ToolContentUnionType = GeometryContentModelType |
                                     TextContentModelType |
                                     UnknownContentModelType;
 
+export type ToolMetadataUnionType = GeometryMetadataModelType;
+
 interface IToolMap {
   [id: string]: IAnyType;
 }
 
 interface IPrivate {
   toolMap: IToolMap;
+  metadataMap: IToolMap;
+  metadata: { [id: string]: ToolMetadataUnionType };
```

Comment:
As we discussed offline, storing `metadata` statically is not ideal, but works while we figure out what's causing the MST reloads